_includes/privacy.html

```diff
@@ -0,0 +1 @@
+<p class="privacy">Please be aware that certain information you provide may not be private. For more information about the privacy of your data, see <a href= "https://redhat-appstudio.github.io/docs.stonesoup.io/Documentation/main/#important-notices-about-the-privacy-of-your-data">Important notices about the privacy of your data.</a></p>
```

Comment:
I think it reads better if we emphasize the importance of this statement. I think the "please be aware" sounds more polite than important. 

What do you think of:

It is important to know that certain information you provide may not be private. For more information about the privacy of your data, see <xref>

Reply:
Privacy note updated.